.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "bracketSpacing": true,
+    "printWidth": 120,
+    "tabWidth": 4,
```

Comment:
How about `"tabWidth": 2`?

Reply:
Then i wait for this change in this pull request. 
Thanks a lot

Reply:
Done!